src/components/select/select.tsx

```diff
@@ -255,11 +255,28 @@ export class Select {
     ) {
         event.stopPropagation();
 
+        const selector = `#${this.portalId} limel-menu-surface limel-list`;
+        const list: HTMLElement = document.querySelector(selector);
+
+        if (!list) {
+            return;
+        }
+
+        const scrollPosition = list.scrollTop;
+        const delay = 8;
+
+        list.scrollTop += 10; // Increase the scroll position by 10
+
         if (isMultiple(event.detail)) {
             const listItems: ListItem[] = event.detail;
             const options: Option[] = listItems.map((item) => item.value);
             this.change.emit(options);
 
+            // Use setTimeout to restore scroll position after a slight delay
+            setTimeout(() => {
+                list.scrollTop = scrollPosition;
+            }, delay);
+
```

Comment:
```suggestion
            requestAnimationFrame(() => {
                list.scrollTop = scrollPosition;
            });
```
We just set the scroll position on the next animation frame.

Reply:
ah!!! so cool, I tried to use `requestAnimationFrame` instead but it didn't work for me, it could be because I missed `ul`..🤔

Reply:
When I first tried `requestAnimationFrame` it didn't work. I think it was adding the `ul` to the selector.